RETROFIT_A_ROLE.md

```diff
@@ -0,0 +1,229 @@
+A brief guide on how to retrofit a role with CI.
+
+TL;DR
+=====
+
+Please, don't make me read all of that down there!
+
+Create Sceanrio
+---------------
+`molecule init scenario -r my_role -s my_scenario`
+
+Configure Scenario Options
+--------------------------
+`vim molecule/my_scenario/molecule.yml` to edit Molecule options
```

Comment:
While you *should* use vim, not everyone does.  Maybe change this to `$EDITOR` or something.

Reply:
Bah humbug. At least I didn't say "nano" :-P